docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -266,16 +216,7 @@
                 Expected 'applied' state but the app is in
                 '{{ app_apply_status.stdout }}' state."
         when: not ("applied" in app_apply_status.stdout)
-
-    # Delete old Deployment Manager pod if it was reinstalled,
-    # the pod to delete may already be terminated by helm
-    - name: Restart Deployment Manager if reinstalled
```

Comment:
A bit concerned about removing this task:

Request test:
1. give a wrong auth password in the system endpoint in the initial deployment, check the DM was blocked by the wrong endpoint password
2. update the password and apply it again. 
The expectation is the DM can get the new password and pass the authentication. As far as I know we'll need to delete the pod once to get the new secret